src/components/professors/ProfList.jsx

```diff
@@ -110,8 +110,8 @@ export default function ProfList() {
       </InputGroup>
 
       {filteredArray.length ? (
-        filteredArray.map((prof) => (
-          <ProfComponent prof={prof} key={prof.tel} />
+        filteredArray.map((prof,index) => (
+          <ProfComponent prof={prof} key={index} />
```

Comment:
avoid using array indexes as keys in such cases. Keys should be persistent between renders. 

`prof.tel` is already a unique value. If i would change it, the only other unique param is `prof.email` which would make more sense as a key

Reply:
Yes, I agree.
I have made the changes to use a concatenation of `prof-email`-`prof.tel`
This will make sure that the key is unique and persistent, even if in case `prof.tel` is not unique.